src/io/jesi/backpack/macros.cljc

```diff
@@ -206,16 +207,16 @@ A single default expression can follow the clauses, and its value will be return
                      ; - https://github.com/reagent-project/reagent/issues/358
                      ; - https://clojure.atlassian.net/browse/CLJS-2710
                      `(~sym [this# ~@args more#]
-                        (if (seq? more#)
-                          (~invoke-fn this# (list* ~@args more#))
-                          (~invoke-fn this# (concat ~args (list more#)))))
+                        (apply ~invoke-fn this# (concat ~args (if (seq? more#)
+                                                                more#
+                                                                (list more#)))))
                      `(~sym [this# ~@args more#]
-                        (~invoke-fn this# (list* ~@args more#))))))]
+                        (apply ~invoke-fn this# (concat ~args more#))))))]
```

Comment:
Tried to concat at compile time, but couldn't get it to work. It doesn't know what more is....